plugin/mru.vim

```diff
@@ -998,6 +1012,8 @@ else
 	\ call s:MRU_Cmd(<q-args>, '', <count>)
 endif
 command! -nargs=0 MruRefresh call s:MRU_Refresh()
+command! -nargs=? -complete=customlist,s:MRU_Complete MRUToggle
+            \ call s:MRU_Toggle(<q-args>)
 
```

Comment:
Thanks for the patch. Can you also update the help text with
a description of this new command?

Reply:
This was my first time actually documenting a vim plugin. I hope this was all that you meant.